UPGRADE_PATH.md

```diff
@@ -1,6 +1,28 @@
 ### Upgrade Paths:
 Use this document to help you understand how to update between major versions of the library. We test upgrade paths when we ramp major versions and do our best to have the compiler help make a useful upgrade experience.
 
+<details>
+  <summary><b>V3 -> V4</b></summary>
+
+  ### Name Change
+  The library changed its name from "Workflow" to "SwiftCurrent". This change was an important step for us, because it helps with SEO and gives people a sense that the library isn't just some generic thing slapped together. 
+
+  **If you're using CocoaPods**
+  Update your `podfile` to include `SwiftCurrent`
+  ```ruby
+  pod 'SwiftCurrent'
```

Comment:
This doesn't align with our installation instructions.  #36 has some renames in the installation instructions... I'm not sure if this should be added to the installation instructions or if we just call out that you should be changing instances of `Workflow` to `SwiftCurrent`?

@morganzellers thoughts?

Reply:
I think calling out to change instances of `Workflow` in the Podfile to `SwiftCurrent` is good, that way it keeps us out of the specification of the `UIKit` vs `Core` usage